ui/src/app/shared/header/header.component.ts

```diff
@@ -272,6 +274,19 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
     this.onDisplayFieldsChanged.emit(this.displayFields);
   }
 
+  isSignedIn(): boolean {
+    return !!this.authService.userId;
```

Comment:
My interpretation of this is
```
!this.authService.userId
```
casts and negates `userId` to `false` if it exists, then the next `!` negates that again.

Is there a way to cast without negating twice, such as (pseudo-code)
```
return (boolean)this.authService.userId
```
?

Reply:
( not a blocker though )

Reply:
This is the way I've seen it done in other parts of the codebase, so I copied that. It looks like `return Boolean(this.authService.userId);` will behave the same. Would that be preferable?

Reply:
Consistency > nitpicking